templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates a Container App.'
```

Comment:
@jongio , I think this is "Creates a container app in an Azure Container App environment." Pls confirm.